src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int width;
+    private final int length;
+
+    public Rectangle(Color color, int sideA, int sideB) {
+        super(color);
+        this.width = sideA;
+        this.length = sideB;
+    }
+
+    @Override
+    public double calculateArea() {
+        return width * length;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Rectangle "
+                + ", area: "
+                + calculateArea() + " sq.units, width: "
+                + width + " units, length: "
+                + length + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
[you need to use method `name()` to get a string representation of the enum. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)
```suggestion
                + getColor().name());
```